buildSrc/src/main/java/Dependencies.kt

```diff
@@ -21,6 +21,7 @@ object Versions {
     const val jna = "5.8.0"
 
     const val androidx_compose = "1.1.1"
+    const val androidx_compose_compiler = "1.2.0-rc02"
```

Comment:
Upgrading to 1.2.0 would involve also upgrading Kotlin.

Reply:
Curious if we have an issue to upgrade Kotlin next so that we can also bump the Compose version

Reply:
We have https://github.com/mozilla-mobile/android-components/issues/12489 for AC. It also involves upgrading Room.